tests/components/bmw_connected_drive/test_config_flow.py

```diff
@@ -110,3 +111,51 @@ async def test_options_flow_implementation(hass: HomeAssistant) -> None:
         }
 
         assert len(mock_setup_entry.mock_calls) == 1
+
+
+async def test_reauth(hass: HomeAssistant) -> None:
+    """Test the reauth form."""
+    with patch(
+        "bimmer_connected.api.authentication.MyBMWAuthentication.login",
+        side_effect=login_sideeffect,
+        autospec=True,
+    ), patch(
+        "homeassistant.components.bmw_connected_drive.async_setup_entry",
+        return_value=True,
+    ) as mock_setup_entry:
+        WRONG_PASSWORD = "wrong"
+
+        FIXTURE_CONFIG_ENTRY_WRONG_PW = deepcopy(FIXTURE_CONFIG_ENTRY)
```

Comment:
Seems like this should just be `fixture_config_entry_wrong_pw` since its not a constant really.

Reply:
Updated as well 👍